<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankExpense</class>
 <widget class="QWidget" name="BankExpense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;生活缴费&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="electricityLabel">
        <property name="text">
         <string>本月应缴电费：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="electricityLineEdit">
        <property name="text">
         <string>114</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="waterLabel">
        <property name="text">
         <string>本月应缴水费：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="waterLineEdit">
        <property name="text">
         <string>50</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="netLabel">
        <property name="text">
         <string>本月应缴网费：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="netLineEdit">
        <property name="text">
         <string>20</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="learnFeeLabel">
        <property name="text">
         <string>本学期应缴学费：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="learnFeeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="residenceLabel">
        <property name="text">
         <string>本学年应缴住宿费：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="residenceLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="commitButton">
        <property name="text">
         <string>一键缴纳</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
